src/vm/interoputil.cpp

```diff
@@ -898,8 +898,8 @@ void FillExceptionData(
 
 //---------------------------------------------------------------------------
 //returns true if pImport has DefaultDllImportSearchPathsAttribute
-//if true, also returns dllImportSearchPathFlag and searchAssemblyDirectory values.
-BOOL GetDefaultDllImportSearchPathsAttributeValue(IMDInternalImport *pImport, mdToken token, DWORD * pDllImportSearchPathFlag)
+//if true, also returns dllImportSearchPathFlags and searchAssemblyDirectory values.
```

Comment:
This 'and searchAssemblyDirectory' phrase should be removed. It is very confusing. The custom attribute value is being returned "unmodified" the comment alludes to some other value that is protected against elsewhere.